docs/user-guide/doc-odm-user-guide/setup.md

```diff
@@ -1,81 +1,229 @@
+   
+3. **Confirm Group Creation:**   
+    * The response will confirm that the group has been successfully created, including details such as the group's ID and creation date.  
+    * Verify that the group is active in ODM by searching for it in the Groups section.  
+    * By default, the user who created the group is the only member and admin. Additional members can be added as needed.
+![16Confirm.png](doc-odm-user-guide/images/16Confirm.png)
+<figcaption>The response will show the group has been successfully created. Corroborate the recently created group, <strong>Demo ODM version 2</strong>, that is now active in ODM by searching for it in the Groups section. By default, the user who created the group is the only active member, but new members can be added</figcaption>
+
+## Managing Groups
+
+Once you have created groups, you can edit details such as members, role members (group admin or group members), and delete groups.
+
+To manage groups in the interface, navigate to the section Groups (instructions described above) and select the group you want to manage.
+
+### Add New Members:
+
+* Click on the **\+New members** button to add members to the group.
+* In the new window, you can select members from the list or use the search bar to find specific users.
+![17Members.png](doc-odm-user-guide/images/17Members.png)
+<figcaption>Add new members to the group by clicking on <strong>+New members</strong>. A new window will appear where you can select users from the list or you can use the search bar to find specific users, e.g., <strong>New User 2024</strong></figcaption>
+
+**Confirm New Members:**
+
+* After selecting the members, click **Add member** to confirm. The group will now display the recently added members.
+![18ConfirmMembers.png](doc-odm-user-guide/images/18ConfirmMembers.png)
+<figcaption>Confirm the new members to add by clicking <strong>Add member</strong>. The recently created group, <strong>Demo ODM</strong> for this example, will now show the recently added member, <strong>New User 2024</strong></figcaption>
+
+### Edit Group Details:
+
+* You can edit the roles of group members by clicking on their role descriptions. You can change members to administrators or vice versa. Note that each group must have at least one administrator.
+* To remove a member, click on the three dots next to their username.
+
+![19Members.png](doc-odm-user-guide/images/19Members.png)
+<figcaption>Edit group members. You can change the roles for the members to either <strong>group member</strong> or <strong>group administrator</strong>. Keep in mind that each group requires at least one admin. You can also remove a member by clicking on the three dots next to the username</figcaption>
+
+## Important Limitations on Group Management
+
+To manage groups (create, edit, or delete), you must have the **Manage Groups** permission. Refer to the Available Permissions section for more information about permissions.
```

Comment:
can we maybe move this information to the beginning of the article to make it clear for users, and make users familiar with this info from the beginning? because I've added several comments regarding cases where permission isn't needed or needed, and removed them after I saw this information at the end of the article